extensions/panache/mongodb-panache-rx/runtime/src/main/java/io/quarkus/mongodb/panache/rx/PanacheMongoEntityBase.java

```diff
@@ -0,0 +1,872 @@
+     * @see #list(String, Sort, Parameters)
+     * @see #stream(String, Sort, Parameters)
+     */
+    @GenerateBridge
+    public static <T extends PanacheMongoEntityBase> CompletionStage<List<T>> list(Document query, Document sort) {
+        throw ReactiveMongoOperations.implementationInjectionMissing();
+    }
+
+    /**
+     * Find all entities of this type.
+     * This method is a shortcut for <code>findAll().list()</code>.
+     *
+     * @return a {@link List} containing all results, without paging
+     * @see #listAll(Sort)
+     * @see #findAll()
+     * @see #streamAll()
+     */
+    @GenerateBridge
+    public static <T extends PanacheMongoEntityBase> CompletionStage<List<T>> listAll() {
+        throw ReactiveMongoOperations.implementationInjectionMissing();
+    }
+
+    /**
+     * Find all entities of this type, in the given order.
+     * This method is a shortcut for <code>findAll(sort).list()</code>.
+     *
+     * @param sort the sort order to use
+     * @return a {@link List} containing all results, without paging
+     * @see #listAll()
+     * @see #findAll(Sort)
+     * @see #streamAll(Sort)
+     */
+    @GenerateBridge
+    public static <T extends PanacheMongoEntityBase> CompletionStage<List<T>> listAll(Sort sort) {
+        throw ReactiveMongoOperations.implementationInjectionMissing();
+    }
+
+    /**
+     * Find entities matching a query, with optional indexed parameters.
+     * This method is a shortcut for <code>find(query, params).stream()</code>.
+     *
+     * @param query a {@link io.quarkus.mongodb.panache query string}
+     * @param params optional sequence of indexed parameters
+     * @return a {@link Stream} containing all results, without paging
+     * @see #stream(String, Sort, Object...)
+     * @see #stream(String, Map)
+     * @see #stream(String, Parameters)
+     * @see #find(String, Object...)
+     * @see #list(String, Object...)
+     */
+    @GenerateBridge
+    public static <T extends PanacheMongoEntityBase> Publisher<T> stream(String query, Object... params) {
+        throw ReactiveMongoOperations.implementationInjectionMissing();
+    }
+
+    /**
+     * Find entities matching a query and the given sort options, with optional indexed parameters.
+     * This method is a shortcut for <code>find(query, sort, params).stream()</code>.
+     *
+     * @param query a {@link io.quarkus.mongodb.panache query string}
+     * @param sort the sort strategy to use
+     * @param params optional sequence of indexed parameters
+     * @return a {@link Stream} containing all results, without paging
+     * @see #stream(String, Object...)
+     * @see #stream(String, Sort, Map)
+     * @see #stream(String, Sort, Parameters)
+     * @see #find(String, Sort, Object...)
+     * @see #list(String, Sort, Object...)
+     */
+    @GenerateBridge
+    public static <T extends PanacheMongoEntityBase> Publisher<T> stream(String query, Sort sort, Object... params) {
```

Comment:
I think that we didn't want to offer a `java.util.Stream` for reactive stream because it will confuse the user as they are lazy by natures so you will end up with stream lazyness and reactive stream asynchronous processing that are also lazyness in a some way.
I would keep it like this or change the method name. But definitly not offering a method that return a `java.util.Stream`.

As always, better propose less then too much as we can add methods later but it's difficult to remove them after some user bet used to them :)